arviz/plots/plot_utils.py

```diff
@@ -579,7 +585,10 @@ def calculate_point_estimate(point_estimate, values, bw="default", circular=Fals
         else:
             point_value = mode(values)[0][0]
     elif point_estimate == "median":
-        point_value = np.median(values)
+        if skipna:
```

Comment:
We should add tests for this specific function as well at the plotting level

Reply:
I have added a test for `skipna` (True or False) for both matplotlib and bokeh. What else is needed?